packages/driver/src/cypress/error_messages.ts

```diff
@@ -1178,7 +1178,9 @@ export default {
       message: stripIndent`
         {{error}}
 
-        Variables must either be defined within the ${cmd('origin')} command or passed in using the args option.`,
+        Variables must either be defined within the ${cmd('origin')} command or passed in using the args option.
+
+        Using \`require()\` or \`import()\` to include dependencies requires using the latest version of \`@cypress/webpack-preprocessor\`.`,
```

Comment:
Suggest specifying the required version of the webpack preprocessor.

Reply:
The trouble is, I won't know what version that is until this is merged and it's actually released. 

Also, when this was originally released, I could have guessed the version was going to be `5.13.0` and hard-coded that into the error message, but webpack-preprocessor@5.13.0 ended being broken. That required a `5.13.1` release, meaning the error message would have been misleading. I think recommending the latest version is the safest bet.